commons/src/main/java/gov/cms/qpp/conversion/model/error/ErrorCode.java

```diff
@@ -128,7 +128,9 @@
 			"`(Measure Id)`'s Sub-population `(Sub Population)`", true),
 	CPC_PLUS_SUPPLEMENTAL_DATA_MISSING_COUNT(67, "Must have one count for Supplemental Data `(Supplemental Data Code)` " +
 			"on Sub-population `(Sub Population)` for eCQM measure `(Measure Id)`", true),
-	CPC_PLUS_SUBMISSION_ENDED(68, "CPC+ Submission is after the end date `(Submission end date)`", true);
+	CPC_PLUS_SUBMISSION_ENDED(68, "CPC+ Submission is after the end date `(Submission end date)`", true),
+	INVALID_PERFORMANCE_PERIOD_FORMAT(69, "`(Performance period start or end date)` is an invalid date format. "
+		+ "Please use the format 'yyyyMMdd'. Ex.: '20170101'", true);
```

Comment:
This message needs to change.  The user just needs to send in some kind of string that's parseable as a date or a date/time.  Maybe change the last bit to "Please use a standard ISO date format".

Reply:
Yeah.  For example, possible valid values are `1980-02-26`, or `1980/02/26T01:45:23` or even `1980-02-26This is one sweet date yo!`.